dpos/log/eventlogs.go

```diff
@@ -9,38 +9,44 @@ type EventLogs struct {
 
 func (e *EventLogs) OnProposalArrived(prop *ProposalEvent) {
 	Info(fmt.Sprintf("[OnProposalArrived] "+
-		"Proposal: %s, "+
+		"Sponsor: %s, "+
```

Comment:
too  much plus operator, can we try bytes.Buffer ?

Reply:
I will check all the event logs

Reply:
sorry, strings.Builder maybe better